lighthouse-cli/chrome-finder.ts

```diff
@@ -75,14 +79,59 @@ export function darwin() {
 }
 
 export function linux() {
+  let installations: Array<string> = [];
+
+  // Look into LIGHTHOUSE_CHROMIUM_PATH
   const execPath = process.env.LIGHTHOUSE_CHROMIUM_PATH;
   if (execPath && canAccess(execPath)) {
-    return [execPath];
+    installations.push(execPath);
   }
-  throw new Error(
-    'The environment variable LIGHTHOUSE_CHROMIUM_PATH must be set to ' +
-    'executable of a build of Chromium version 52.0 or later.'
-  );
+
+  // Search in desktop files for executable
+  const desktopInstallationFolders = [
+    process.env.HOME + '/.local/share/applications/',
+    '/usr/share/applications/',
+  ];
+  desktopInstallationFolders.forEach(folder => {
+    installations = installations.concat(findChromeExecutables(folder));
+  });
+
+  // Search for executable with which command
+  const executables = [
+    'google-chrome-stable',
+    'google-chrome',
+  ];
+
+  executables.forEach((executable: string) => {
+    const chromePath = execSync(`which ${executable}`)
+      .toString()
+      .split(newLineRegex)[0];
+
+    if (chromePath && canAccess(chromePath)) {
+      installations.push(chromePath);
+    }
+  });
+
+  if (!installations.length) {
+    throw new Error('The environment variable LIGHTHOUSE_CHROMIUM_PATH must be set to ' +
+      'executable of a build of Chromium version 52.0 or later.');
+  }
+
+  const priorities: Priorities = [{
+    regex: new RegExp(`chrome-wrapper$`),
```

Comment:
LIGHTHOUSE_CHROMIUM_PATH gets top priority. As a second we give chrome-wrapper (chromium) the next priority in the list and as last we have chrome. (should I document it?)